src/npmlicensecheck.ts

```diff
@@ -0,0 +1,58 @@
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+import * as prcomment from "./prcomments";
+
+const blockedLicenses = core.getMultilineInput("blockedLicenses");
+const continueOnBlockedFound = core.getBooleanInput("continueOnBlockedFound");
+
+const processNpm = async (projectPath: string, pullRequestNumber: number): Promise<void> => {
+  core.info(`Starting processNpm for: ${projectPath}`);
+
+  await exec.exec("yarn", [""], {
+    silent: true,
+  });
+
+  const { stdout: licenseReport } = await exec.getExecOutput(
+    "npx",
```

Comment:
if you are using npx you don't have to install the npm package globally like you did in the index.ts file :-) 

I guess the question is if you just always want to use the latest version (with npx) or if you want to use a specific version? 

pros of npx is you always have the latest.
cons of npx is that they can release a new version which will brake this package.